docs/configuration/appointment.rst

```diff
@@ -0,0 +1,51 @@
+.. _configuration_appointment:
+
+=========================
+Appointment configuration
+=========================
+
+Appointments can be made using two separate services.  JCC and Qmatic.
+Open Forms can be configured to make use of one of these services.
+
+JCC
+---
+
+1. In Open Forms navigate to: **Configuration** > **JCC configuration**
+2. Click the **Green Plus Button** and fill in the following details:
+
+   * **Label**: JCC
+   * **Ontvangende organisatie**: Open Forms
+   * **Ontvangende applicatie**: Open Forms
+   * **Versturende organisatie**: Horstaandemaas
+   * **Versturende applicatie**: Afspraak Acceptatie
+   * **URL**: https://afspraakacceptatie.horstaandemaas.nl/JCC/Horst%20aan%20de%20Maas/WARP/GGS2/GenericGuidanceSystem2.asmx?wsdl
```

Comment:
This is what I've used for testing JCC but I'm not sure if we want to mention something different here?  Also for this particular endpoint I don't think the Ontvangende/Versturende fields actually matter since I tried a few different values with testing this and everything seemed to work.

Reply:
Stop, remove that URL please!

Please use: `https://example.com/jcc/GenericGuidanceSystem2.asmx?wsdl`

Reply:
Stop, remove that URL please and rebase as if it was never here!

Please use: `https://example.com/jcc/GenericGuidanceSystem2.asmx?wsdl`